documentation/modules/configuring/con-config-mirrormaker2.adoc

```diff
@@ -76,97 +76,121 @@ For more information, see xref:con-high-volume-config-properties-{context}[Handl
 .Example `KafkaMirrorMaker2` custom resource configuration
 [source,yaml,subs="+attributes"]
 ----
+# Basic configuration (essential)
 apiVersion: {KafkaMirrorMaker2ApiVersion}
-kind: KafkaMirrorMaker2
+kind: KafkaMirrorMaker2 # <1>
 metadata:
   name: my-mirror-maker2
+# Deployment specifications
 spec:
+  # Kafka version (essential)
   version: {DefaultKafkaVersion} # <1>
+  # Replicas (essential)
   replicas: 3 # <2>
+  # Connect cluster name (essential)
   connectCluster: "my-cluster-target" # <3>
+  # Cluster configurations (essential)
   clusters: # <4>
-  - alias: "my-cluster-source" # <5>
-    authentication: # <6>
-      certificateAndKey:
-        certificate: source.crt
-        key: source.key
-        secretName: my-user-source
-      type: tls
-    bootstrapServers: my-cluster-source-kafka-bootstrap:9092 # <7>
-    tls: # <8>
-      trustedCertificates:
-        - pattern: "*.crt"
-          secretName: my-cluster-source-cluster-ca-cert
-  - alias: "my-cluster-target" # <9>
-    authentication: # <10>
-      certificateAndKey:
-        certificate: target.crt
-        key: target.key
-        secretName: my-user-target
-      type: tls
-    bootstrapServers: my-cluster-target-kafka-bootstrap:9092 # <11>
-    config: # <12>
-      config.storage.replication.factor: 1
-      offset.storage.replication.factor: 1
-      status.storage.replication.factor: 1
-    tls: # <13>
-      trustedCertificates:
-        - pattern: "*.crt"
-          secretName: my-cluster-target-cluster-ca-cert
+    - alias: "my-cluster-source" # <5>
+      # Authentication (recommended)
+      authentication: # <6>
+        certificateAndKey:
+          certificate: source.crt
+          key: source.key
+          secretName: my-user-source
+        type: tls
+      bootstrapServers: my-cluster-source-kafka-bootstrap:9092 # <7>
+      # TLS configuration (optional)
```

Comment:
Same as before, I would say that both `authentication` and `tls` should be the the same.